README.adoc

```diff
@@ -23,4 +23,28 @@ Then write 2 for hdmi_group and 27 for mode for a result of:
 hdmi_group=2 
 hdmi_mode=27 
 . Save the file and reboot the pi.
+ 
+= Compiling and Running the Program
+1. After cloning the repository, navigate to the installer directory: `cd Installer` 
+2. Run the `MainInstaller.sh` script as root: `sudo ./MainInstaller.sh`
+3. Generate the executable with `make -j4`, this will make it a lot faster as it is generated using 4 cores
```

Comment:
` this will make it a lot faster` is a little bit too informal haha, maybe just say. `.... with make -j4. -j4 will decrease the compile time by using 4 threads, this number can be increased` (I was actually wrong the first time I mentioned it to you, check this out: https://stackoverflow.com/questions/15289250/make-j4-or-j8)